Structs/__init__.py

```diff
@@ -211,7 +224,8 @@ def new(cls: Type[NamedTuple], *args: Any, **kwargs: Any) -> NamedTuple:
 
         # Expand any nested fstructs
         args = [
-            convert_arg(val, idx)
+            val if idx > len(fields) else
+            convert_arg(val, fields[idx])
```

Comment:
If there are more args than fields, it's an error anyway - might as well just let the implicit index error fire and keep our code simpler